server/app/modules/join_hunt/router.py

```diff
@@ -0,0 +1,17 @@
+from fastapi import APIRouter
+from app.modules.join_hunt.service import service
+from app.modules.join_hunt.router_models import Join_Hunts_Model
+
+router = APIRouter()
+
+@router.post(
```

Comment:
By the way, you should add documentation describing error responses to this endpoint. Something like this:

```
@router.put(
  "/{id}",
  status_code = 200,
  response_model=UserResponseModel,
  responses={
    404: {
      "description": "User not found",
      "model": UserResponseModel
    },
    422: {
      "description": "Request could not be validated",
      "model": ValidationErrorsModel
    }
  }
)
```

This depends on what errors your endpoint could return. If there is currently a "422" responser written in the docs, make sure to override it with the one above, as that is the correct response in the case of 422 errors.